.github/workflows/gh-ci.yaml

```diff
@@ -88,6 +88,12 @@ jobs:
       with:
         build-tests: true
         build-docs: false
+        isolation: ${{ matrix.numpy != '' }}
```

Comment:
if we define an old numpy value then we isolate (that way we can build with the correct numpy version and then test with a runtime version of numpy that's much older)